src/anomalib/utils/visualization/image.py

```diff
@@ -102,7 +102,8 @@ def _visualize_batch(self, batch: dict) -> Iterator[GeneratorResult]:
         for i in range(batch_size):
             if "image_path" in batch:
                 height, width = batch["image"].shape[-2:]
-                image = read_image(path=batch["image_path"][i], image_size=(height, width))
+                image = read_image(path=batch["image_path"][i])
+                image = cv2.resize(image, dsize=(width, height), interpolation=cv2.INTER_AREA)
```

Comment:
would it be an idea to keep the resize option in `read_image`. So we would not use cv2?

With PIL we could do it with `image.resize(new_size)` in `read_image`?

Reply:
Let's address this later.